Inventory/IInventory.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace InventorySystem.Inventory
+{
+    public interface IInventory
+    {
+        void AddProduct(Product product);
+
+        bool Exists(string name);
+
+        Product? GetProduct(string name);
+
+        void PrintInventory();
+
+        void RemoveProduct(Product? product);
+
+        void EditProduct(Product product, String? name, String? Description, double Price);
+    }
+}
```

Comment:
Consider using file-scoped names spaces. 

```suggestion
﻿namespace InventorySystem.Inventory;
﻿
public interface IInventory
{
    void AddProduct(Product product);

    bool Exists(string name);

    Product? GetProduct(string name);

    void PrintInventory();

    void RemoveProduct(Product? product);

    void EditProduct(Product product, String? name, String? Description, double Price);
}
```